sway-ir/src/optimize/simplify_cfg.rs

```diff
@@ -0,0 +1,150 @@
+use crate::{
+    block::Block, context::Context, error::IrError, function::Function, instruction::Instruction,
+    value::ValueDatum,
+};
+
```

Comment:
could we get a high level description of how simplify cfg works at the top of this module?